src/components/Button/Button.stories.tsx

```diff
@@ -195,7 +195,7 @@ stories
   ))
   .add("Button Group", () => {
     // props which should be same for all Buttons in ButtonGroup
-    const type = options("type", typeOptions, "primary", {
+    const type = options("type", kindOptions, "primary", {
```

Comment:
we have to change type here also, Buttons used in ButtonGroup has to change attribute from `type` to `kind`.